src/diagnostics/structs/Logpoint.ts

```diff
@@ -0,0 +1,8 @@
+export class Logpoint {
```

Comment:
this could just be a interface & things would be more flexible.

Reply:
Done